### PR TITLE
Combined login signup

### PR DESCRIPTION
**Pull request by michaeleggboy**
Note: Login pull request essential the same thing but w/o combined login-signup page, we don't need to merge both. Look at login pull request for more screenshots.

# Change Log
src/index.css: added .center-tabs class that centers tabs in the middle of the page and also fixes the dimensions regardless of feedback. 
src/components/Auth/LoginPage/index.js: added login form with needed input functionality
src/components/Auth/SignUpPage/index.js: added registration form with needed input functionality
src/components/Auth/GetStartedPage/index.js: added switch tabs to access login and signup page without redirecting 
src/App.js: deleted /login and /sign-up routing, added /get-started path (this is important to note for the home page's login/sign-up buttons, should replace with a singular get started button)

# Screenshots

## New
Screenshot 2020-06-29 at 9 07 49 AM
Screenshot 2020-06-29 at 9 07 43 AM

## Old
Screenshot 2020-06-26 at 7 07 13 PM
Screenshot 2020-06-26 at 7 07 22 PM

# Next Steps
src/components/Auth/LoginPage/index.js: add submit functionality to write and read into a database
src/components/Auth/SignUpPage/index.js: add submit functionality to write and read into a database
src/components/Auth/GetStartedPage/index.js: add more cosmetic styling

### CODE REVIEW

File: react-app/src/components/Auth/LoginPage/index.js [hunk 1]
```diff
@@ -1,5 +1,64 @@
...
+import '../../../index.css';
+import { Form, Input, Button, Checkbox } from 'antd';
+import { UserOutlined, LockOutlined } from '@ant-design/icons';
 
-const LoginPage = () => <div>LoginPage</div>
+const LoginPage = () => {
+  
+  const onFinish = values => {
+    console.log('Received values of form: ', values);
+  };
```
Comment (clairezc27) [inline react-app/src/components/Auth/LoginPage/index.js:11]: everything looks great! I'm assuming you need these console logs for later?

Comment (michaeleggboy) [reply react-app/src/components/Auth/LoginPage/index.js:11]: we can removed them later when we have actually functionality for submit but it currently is just there to make sure the form and button work

**Comment by michaeleggboy** [comment]
> oops i approved the other one

fixed it

**Comment by alisa-lin** [comment]
@marsw42

**Comment by michaeleggboy** [comment]
> Pull in master to remove all build files

It tells me that's up to date

**Comment by michaeleggboy** [comment]
@marsw42 removed build file

**Pull request merged**